Views/Branches/Create.cshtml

```diff
@@ -0,0 +1,89 @@
+﻿@model StoresManagement.ViewModels.BranchFormViewModel
+
+@{
+    ViewData["Title"] = "Create";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h1>Create</h1>
+
+<h4>Branch</h4>
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        <form asp-action="Create">
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <div class="form-group">
+                <label asp-for="EntityId" class="control-label"></label>
+                <select asp-for="EntityId" class="form-control" asp-items="ViewBag.EntityId"></select>
+            </div>
+            <div class="form-group">
+                <label asp-for="Identification" class="control-label"></label>
+                <input asp-for="Identification" class="form-control" />
+                <span asp-validation-for="Identification" class="text-danger"></span>
```

Comment:
decide and keep consistient with convention. In `Views/Branches/Delete.cshtml` you use helper methods.